src/app/tabs/run-tab.js

```diff
@@ -264,12 +274,12 @@ function contractDropdown (appAPI, appEvents, instanceContainer) {
       </div>
       <div class="${css.buttons}">
         <div class="${css.button}">
-          <div class="${css.atAddress}" onclick=${function () { loadFromAddress(appAPI) }}>At Address</div>
-          ${atAddressButtonInput}
+        ${createButtonInput}
```

Comment:
why having changed the order?

Reply:
if there's no special reason, could you keeping the same order as before?

Reply:
I was thinking that Create instance is more common action, so I put in on top. But right now I am thinking we need to improve this whole thing again. I think we should also have a dropdown here. So `create` by default, but if you want, you can also `load`. 

Because it's either-or, so dropdown makes more sense. Otherwise it's a confusion like @holgerd77 already stated yesterday that users don't know if they need to press both buttons or what needs to be done here...

I will make necessary changes and then you can review and see if it works. I think the UI logic is way better this way.